map_claims.go

```diff
@@ -126,20 +126,17 @@ func (m MapClaims) Valid() error {
 	now := TimeFunc().Unix()
```

Comment:
You need to remove the unused import to `errors` as well. You do not need to open/close a new PR, you can add new commits to your existing branch.

Reply:
All right, handled